src/DCMTrajectoryGeneratorHelper.cpp

```diff
@@ -407,6 +407,7 @@ bool DoubleSupportTrajectory::getZMPPosition(const double &t, iDynTree::Vector2
 
 DCMTrajectoryGeneratorHelper::DCMTrajectoryGeneratorHelper():
     m_dT(0.01),
+    m_alpha(0.5),
```

Comment:
Please, place this after ``m_omega``. The initialization order should follow the declaration order in the ``.h`` file, otherwise the compiler throws a warning.

Reply:
Ok. I also changed the order of the  `m_pauseActive(false)` that already had some warning:
https://github.com/robotology/unicycle-footstep-planner/blob/5c3f6d8ca9f814bd2efe21cbf47feb3c7d80ec70/src/DCMTrajectoryGeneratorHelper.cpp#L413

Reply:
Great! Thanks